docs/docs-beta/docs/guides/deployment/kubernetes.md

```diff
@@ -1,4 +1,215 @@
+- How to access and customize the Dagster-provided Helm charts
+- How to apply the configuration in the Helm charts to your Kubernetes cluster to deploy your Dagster project
+- How to connect to your deployed Dagster project
+
+<details>
+  <summary>Prerequisites</summary>
+
+To follow the steps in this guide, you'll need:
+
+- Familiarity with [Docker](https://docs.docker.com/)
+- Familiarity with [Kubernetes](https://kubernetes.io/docs/home/)
+- Familiarity with [Helm](https://helm.sh/docs/)
+- A Dagster project to deploy. You can also use the [example project](/todo). You can download the example project using the command:
+```bash
+dagster project from-example --example deploy_k8s_beta --name assets_dbt_python
+```
+- To have Docker installed. [Docker installation guide](https://docs.docker.com/engine/install/)
+- To have `kubectl` installed. [Kubernetes installation guide](https://kubernetes.io/docs/tasks/tools/)
+- To have a Kubernetes cluster created. If you would like to follow along with this guide on your local machine, you can install Docker Desktop and turn on the included Kubernetes server [Docker Desktop and Kubernetes guide](https://docs.docker.com/desktop/kubernetes/)
+- Access to a Docker image registry, such as Amazon Web Services ECR or DockerHub. If you are following along to the guide on your local machine, you do not need access to a Docker image registry.
+- To have Helm 3 installed. [Helm installation guide](https://helm.sh/docs/intro/install/)
+
+</details>
+
+## Step 1: Get your Dagster project ready to deploy
+You will need to add a `workspace.yaml` file to your Dagster project to be ready to deploy with Kubernetes. If you are using the [example project](/todo), you will already have this file.
+
+The `workspace.yaml` file tells Dagster where to find the `Definitions` object in your project. This file should be at the LOCATION of your project.
+
+<CodeExample filePath="guides/deployment/kubernetes/workspace.yaml" language="yaml" title="Example workspace.yaml" />
+For example, in the sample `iris_analysis` project, the `Definitions` object is found in the `iris_analysis/__init__.py` file. The `workspace.yaml` tells Dagster to load this project and give it the name `iris_analysis`.
+
+You can also point to a Python module in the `workspace.yaml` file. To learn more about writing a `workspace.yaml` see the [Workspace](/todo) guide.
+
+## Step 2: Write and build a Docker image containing your Dagster project
+### Step 2.1: Write a Dockerfile
+You will need to build a Docker image that contains your Dagster project and all of its dependencies. The Dockerfile should copy your Dagster project and the `workspace.yaml` file created in Step 1 into the image and install `dagster`, `dagster-postgres`, and `dagster-k8s`, along with any other libraries your project depends on. Finally, ensure that port 80 is exposed. This will be used to set up port-forwarding later.
```

Comment:
I think this is a lot of dense information - thoughts on breaking it up a little? Ex:

```suggestion
Next, you'll build a Docker image that contains your Dagster project and all of its dependencies. The Dockerfile should:

1. Copy your Dagster project and the `workspace.yaml` file into the image
2. Install `dagster`, `dagster-postgres`, and `dagster-k8s`, along with any other libraries your project depends on. The example project has a `pandas` dependency, so it's included in `pip install` in the following example `Dockerfile`.
3. Expose port `80`, which we'll use to set up port-forwarding later in the guide.
```

Reply:
Also:

> The Dockerfile should copy your Dagster project and the `workspace.yaml` file

Do we need to specifically include `workspace.yaml` here? Isn't it part of the Dagster project?

Reply:
What about 
> Copy your Dagster project, including the `workspace.yaml` file created in Step 1, into the image.

Reply:
Works for me!